### ISSUE TITLE
Develop synthetic benchmark models

### ISSUE THREAD

**Issue by paulromano** [openmc-dev/fes-performance-project#2]

**Comment by pshriwise** [comment]
@paulromano I believe you generated a nice tetrahedral model for the mesh volume calculation feature. Would that be a good candidate as well?

**Comment by paulromano** [comment]
Yup, @amandalund and I just synced up on this task and I mentioned [that model](https://github.com/paulromano/fed-material-mesh/blob/main/models/random/generate_random_model.py), which would work well for the "complex geometry with 1000s of cells" after some modifications

**Comment by paulromano** [comment]
Needs for photon transport:
- Stress different energy regimes dominated by photoelectric, Compton scattering, pair production
- Testing thick-target bremsstrahlung
- Testing with and without atomic relaxation (need a setting for this in OpenMC)

**Comment by paulromano** [comment]
For thermal scattering:
- Point source at say 1 eV in a large sphere of a single material
- Materials:
  - H2O (incoherent inelastic)
  - Graphite (coherent elastic)
  - ZrH (coherent inelastic)

**Comment by paulromano** [comment]
For coupled neutron-photon, we want to create a problem that is heavy on photon production from neutron reactions. To do this, we can have a neutron source with a few discrete energies (using `openmc.stats.Discrete`) that correspond to strong resonances in Gd:
- 2.568 eV = strong (n,γ) resonance in Gd155 
- 33.23 eV = strong (n,γ) resonance in Gd156 
- 16.77 eV = strong (n.γ) resonance in Gd157
- 22.3 eV = strong (n,γ) resonance in Gd158
- 222 eV = strong (n,γ) resonance in Gd160
